prow/kube/client.go

```diff
@@ -72,9 +72,21 @@ func (c *Client) log(methodName string, args ...interface{}) {
 	c.Logger.Debugf("%s(%s)", methodName, strings.Join(as, ", "))
 }
 
-type ConflictError error
+type ConflictError struct {
+	E error
```

Comment:
Can you unexport these?

Reply:
Unexported and added constructors instead.